src/sdk/main/include/TokenRelationship.h

```diff
@@ -54,10 +68,26 @@ class TokenRelationship
                     const std::string& symbol,
                     uint64_t balance,
                     uint32_t decimals,
-                    std::string_view kycStatus,
-                    std::string_view freezeStatus,
+                    int kycStatus,
+                    int freezeStatus,
                     bool automaticAssociation);
 
+  /**
+   * Construct a TokenRelationship object from a TokenRelationship protobuf object.
+   *
+   * @param proto The TokenRelationship protobuf object from which to construct an TokenRelationship
+   * object.
```

Comment:
nit: can you tab this over to align with the parameter description?

Reply:
Didn't get what you mean here.

Reply:
I was asking to tab over "object." here so that it aligns with the beginning of the parameter description on the line above it ("The TokenRelationship.."). Helps with readability (which since this is the only parameter I guess it doesn't really improve much so take it or leave it, but in the interest in keeping things in the codebase consistent), other parameter descriptions are like this